guides/Compiling.md

```diff
@@ -79,11 +79,20 @@ using one of those builds.
 Linux
 ---
 
+### Using Docker
+
+If you prefer you can build the application using a Docker image that contains the required packages and components. You can write your code on your host machine, and the container will be used only to build the application.
+
+To use it, you'll need to [install Docker](https://docs.docker.com/engine/install/).
+
+Afterwards, the relevant files and scripts can be found at the [lovr-docker-builder](https://github.com/Udinanon/lovr-docker-builder) project.
+
+
 First, install a C compiler, CMake, and other dependencies:
 
 ### Debian/Ubuntu
 
-    $ sudo apt install cmake clang xorg-dev libcurl-dev
+    $ sudo apt install cmake clang xorg-dev build-essential libcurl4-openssl-dev libxcb-glx0-dev libx11-xcb-dev glslang-tools python3-minimal
```

Comment:
- `glslang` does not need to be installed to build LÖVR for Linux.  Since we have to build glslang from source anyway, we use that to compile shaders.  Android is another story, but that's already documented in the Android section.
- I don't think it makes sense to include both `build-essential` and `clang` since this installs 2 different C compilers.
- Yes, we should change `libcurl-dev` to `libcurl4-openssl-dev`, since `libcurl-dev` is a virtual package.
- Since python isn't required to build anything directly in LÖVR and it's already installed by clang, I think it would be better to leave it out.

Reply:
I think python is needed somewhere if you build it from scratch, i think something in Vulkan? i can't build it in docker unless i install it
`clang` i just took from the docs, but build essentials does something to inform CMake of the available compilers, maybe this can be done in some other way
I need to check with `glslang` cause it didn't work without it I think, at least in docker

Maybe the differences come from a full system VS what the image has?

Reply:
Hm maybe if we only install clang then we are missing a C++ compiler (build-essential installs g++), I'll try it on a fresh Ubuntu.

Reply:
Ok i think build-essentials can be skipped if we set the CC and CXX variables, that can be done in the dockerfile, we only need to also install make
python3 is required at cpmpilation by glslang actually
```
CMake Error at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Python3 (missing: Python3_EXECUTABLE Interpreter)
Call Stack (most recent call first):
  /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.22/Modules/FindPython/Support.cmake:3180 (find_package_handle_standard_args)
  /usr/share/cmake-3.22/Modules/FindPython3.cmake:490 (include)
  deps/glslang/CMakeLists.txt:197 (find_package)
  deps/glslang/StandAlone/CMakeLists.txt:34 (find_host_package)


-- Configuring incomplete, errors occurred!
```

glslang does not need to be isntalled, I'll remove it